docs/user-documentation/objects.md

```diff
@@ -93,3 +93,21 @@ All nodes can contain any number of media.  The media for any node can be manage
 the "Members" tab, Actions can be performed in bulk using the check boxes and Actions drop down.
 
 ![Media tab](../assets/islandora_8_derivatives_sample.png)
+
+## Display Hints
+
+Media are displayed in Islandora 8 via "view modes". View modes are Drupal's name for alternative ways to present content to users. In Islandora, the view mode determines whether a media is displayed using the OpenSeadragon viewer or the PDFjs viewer, for example. View modes can be set explicitly in the node edit form, or you can configure Islandora to use a specific view mode for all media with a specific Mime type.
+
+At the node level, in the "Display hints" field, simply check the display mode you want to use for that node:
+
+![Display hints](../assets/display_hints.png)
+
+The selected view mode will then be used when the node is viewed.
+
+At a global level, you can create a [Context](context.md) that will detect the media's Mime type and use the configured view mode automatically. To do this, go to Drupal's "Structure" menu, then "Context" and create a new Context. Then, add a "Media has Mime type" condition and specify the Mime type, and add a "Change View mode" Reaction that selects the desired view mode:
+
+![Display hints](../assets/view_mode_context.png)
+
+Finally, save your Context. From that point on, whenever the media for a node has the configured Mime type, Drupal will render the media using the corresponding view mode.
+
+The node-level and global approaches are not exclusive to one another. You can configure the view mode globally and override it at the node level if you want.
```

Comment:
This isn't entirely correct as one Context can override the other depending the order of execution. Whichever condition is checked last between the "node-level" condition (has-term-based) the "global" (mime-type-based) will override the other.

Reply:
Yes, I avoided the ordering topic intentionally so as not to get too far into the weeds. But, if we think it's important to document that here, I will do so.

Reply:
I do think it is important to bring up if you have multiple contexts that impact the same display view mode. ([Related comment](#discussion_r305005767).)

Reply:
OK, addressed.